CHANGELOG.md

```diff
@@ -129,7 +132,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ## 1.0.0 - 2019-07-31 [YANKED]
 
 
-[Unreleased]: https://github.com/miaowware/qrm2/compare/v2.4.1...HEAD
+[Unreleased]: https://github.com/miaowware/qrm2/compare/v2.4.2...HEAD
```

Comment:
```suggestion
[Unreleased]: https://github.com/miaowware/qrm2/compare/v2.4.1...HEAD
```
should not have been changed

Reply:
Sure. How is this determined anyways?

Reply:
it's done manually at release time currently

Reply:
Thanks.